config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-config.yaml

```diff
@@ -143,7 +147,8 @@ presubmits:
           # during the tests more like 3-20m is used
           cpu: 2000m
   - name: pull-kubernetes-csi-csi-driver-host-path-alpha-1-13-on-kubernetes-1-13
-    always_run: true
+    always_run: false
+    optional: true
```

Comment:
Just to recap, because we are using the latest code for the hostpath driver, alpha features on 1.13 might break.

Makes sense to allow breaking this configuration.

But should we perhaps nevertheless run this by default (`always_run: true`)? The `optional: true` will then allow merging even when there is a failure.

Without pull jobs which test alpha features, most developers will not get immediate feedback when they are accidentally breaking an alpha feature.

Reply:
I also don't want to give the impression that we must preserve backwards compatibility for alpha features by automatically running a failing test.  We have CI jobs that can cover alpha features after the fact.  I think we have a low amount of alpha features that it should not be difficult for reviewers to remember to trigger an alpha job.  Or maybe for entire alpha sidecars like snapshotter and resizer, we can run alpha by default there.

I was considering removing the alpha pull jobs entirely but decided to keep them in so that we can run alpha testing on older release branches where we shouldn't be breaking across patches.

Reply:
> I also don't want to give the impression that we must preserve backwards compatibility for alpha features by automatically running a failing test.

I'm not convinced. Breaking an alpha feature has pretty severe consequence (must start a new release train, with all the overhead for backporting and testing that this implies), so I think it would make sense to flag such breakage early. Just my 2 cents.

> I think we have a low amount of alpha features that it should not be difficult for reviewers to remember to trigger an alpha job.

I'm a bit more pessimistic here. This puts a lot of faith into reviewers (must recognize that a change might break an alpha feature, must remember that we have Prow jobs for them and what they are named).

Having said that, let's merge this.

/lgtm